src/algorithms/ctmrg/ctmrg.jl

```diff
@@ -92,15 +92,15 @@ const SequentialCTMRG = CTMRG{:sequential}
 const SimultaneousCTMRG = CTMRG{:simultaneous}
 
 """
-    MPSKit.leading_boundary([envinit], state, alg::CTMRG)
+    leading_boundary([envinit], state, alg::CTMRG)
```

Comment:
I know that. Here I just think we should use an independent function.

Reply:
Is there a reason you want them to be independent? They both mean the same thing, and otherwise you would create name clashes if someone imports both MPSKit and PEPSKit. Given how closely related these things are, I think it is valid to give them the same function, but I am open to suggestions